contracts/marker/src/contract.rs

```diff
@@ -206,410 +240,400 @@ fn try_transfer(
-
-        // Assert the correct params were created
-        match unwrap_marker_params(&res.messages[0].msg) {
-            MarkerMsgParams::CancelMarker { denom } => assert_eq!(denom, "budz"),
-            _ => panic!("expected marker cancel params"),
-        }
-    }
-
-    #[test]
-    fn destroy_marker() {
-        // Create default provenance mocks.
-        let mut deps = mock_dependencies(&[]);
-        let env = mock_env();
-        let info = mock_info("sender", &[]);
-
-        // Create a destroy marker handler message
-        let msg = ExecuteMsg::Destroy {
-            denom: "budz".into(),
-        };
-
-        // Call handle and ensure a cosmos message was dispatched
-        let res = execute(deps.as_mut(), env, info, msg).unwrap();
-        assert_eq!(1, res.messages.len());
-
-        // Assert the correct params were created
-        match unwrap_marker_params(&res.messages[0].msg) {
-            MarkerMsgParams::DestroyMarker { denom } => assert_eq!(denom, "budz"),
-            _ => panic!("expected marker destroy params"),
-        }
-    }
-
-    #[test]
-    fn transfer_coins() {
-        // Create default provenance mocks.
-        let mut deps = mock_dependencies(&[]);
-        let env = mock_env();
-        let info = mock_info("sender", &[]);
-
-        // Create a transfer execute message
-        let msg = ExecuteMsg::Transfer {
-            amount: Uint128::new(20),
-            denom: "budz".into(),
-            to: "toaddress".into(),
-        };
-
-        // Call execute and ensure a cosmos message was dispatched
-        let res = execute(deps.as_mut(), mock_env(), info, msg).unwrap();
-        assert_eq!(1, res.messages.len());
-
-        // Assert the correct params were created
-        let expected_coin = coin(20, "budz");
-        match unwrap_marker_params(&res.messages[0].msg) {
-            MarkerMsgParams::TransferMarkerCoins { coin, to, from } => {
-                assert_eq!(*coin, expected_coin);
-                assert_eq!(*to, Addr::unchecked("toaddress"));
-                assert_eq!(from, &env.contract.address);
-            }
-            _ => panic!("expected marker transfer params"),
-        }
-    }
-
-    #[test]
-    fn query_marker() {
-        // Create a mock querier with our expected marker.
-        let bin = must_read_binary_file("testdata/marker.json");
-        let expected_marker: Marker = from_binary(&bin).unwrap();
-        let mut deps = mock_dependencies(&[]);
-        deps.querier.with_markers(vec![expected_marker.clone()]);
-        // Query and ensure we got the expected marker
-        let req = QueryMsg::GetByDenom {
-            denom: "nugz".into(),
-        };
-        let bin = query(deps.as_ref(), mock_env(), req).unwrap();
-        let marker: Marker = from_binary(&bin).unwrap();
-        assert_eq!(marker, expected_marker);
-        assert_eq!(marker.address, "tp18vmzryrvwaeykmdtu6cfrz5sau3dhc5c73ms0u")
-    }
-}
+//
+// #[cfg(test)]
```

Comment:
Same as above.

Reply:
most are back now, but will probably catch the last contract or 2 in the next RC